impeller/aiks/trace_serializer.cc

```diff
@@ -51,11 +51,43 @@ std::ostream& operator<<(std::ostream& os,
   return os;
 }
 
+std::ostream& operator<<(std::ostream& os, const ColorSource& color_source) {
+  os << "{ type: ";
+  switch (color_source.GetType()) {
+    case ColorSource::Type::kColor:
+      os << "kColor";
+      break;
+    case ColorSource::Type::kImage:
+      os << "kImage";
+      break;
+    case ColorSource::Type::kLinearGradient:
+      os << "kLinearGradient";
+      break;
+    case ColorSource::Type::kRadialGradient:
+      os << "kRadialGradient";
+      break;
+    case ColorSource::Type::kConicalGradient:
+      os << "kConicalGradient";
+      break;
+    case ColorSource::Type::kSweepGradient:
+      os << "kSweepGradient";
+      break;
+    case ColorSource::Type::kRuntimeEffect:
+      os << "kRuntimeEffect";
+      break;
+    case ColorSource::Type::kScene:
+      os << "kScene";
+      break;
+  }
+  os << "}";
```

Comment:
```suggestion
  os << " }";
```

Assuming that you wanted it to read: `{ type: kScene }` and not `{ type: kScene}`.